File: test_data/golang/test_main/aas_core_meta.v3/input/snippets/Verification/is_xs_date_time_UTC.go
// Check that `value` is a valid `xs:dateTime` with
// the time zone set to UTC.
func IsXsDateTimeUTC(value string) bool {
	if !MatchesXsDateTime(value) {
		return false
	}

	date, _, ok := strings.Cut(value, "T")
	if !ok {
		panic(
			fmt.Sprintf(
				"Expected 'T' in the date-time if it matches the expected regex, " +
				"but got: %s",
				value,
			),
		)
	}
	return IsXsDate(date)
}

File: test_data/java/test_main/aas_core_meta.v3/input/snippets/Verification/is_xs_date_time_UTC.java
/**
* Check that value is a xs:dateTime with
* the time zone set to UTC.
* @param value the value to check
*/
public static boolean isXsDateTimeUtc(
        String value
){
    if (!matchesXsDateTimeUtc(value))
    {
        return false;
    }

    return isPrefixedWithValidDate(value);
}